tool/test/org/antlr/v4/test/runtime/java/Java.test.stg

```diff
@@ -148,6 +110,44 @@ public void test<test.name>() throws Exception {
 
 >>
 
+testAnnotations(test) ::= <%
```

Comment:
:question: is this a straight copy-paste or should I be looking at content as well?